src/login/logind-button.h

```diff
@@ -5,6 +5,22 @@ typedef struct Button Button;
 
 #include "logind.h"
 
+typedef enum ButtonModifierMask {
+        BUTTON_MODIFIER_NONE        = 0,
+        BUTTON_MODIFIER_LEFT_SHIFT  = 1 << 0,
+        BUTTON_MODIFIER_RIGHT_SHIFT = 1 << 1,
+        BUTTON_MODIFIER_LEFT_CTRL   = 1 << 2,
+        BUTTON_MODIFIER_RIGHT_CTRL  = 1 << 3,
+        BUTTON_MODIFIER_LEFT_ALT    = 1 << 4,
+        BUTTON_MODIFIER_RIGHT_ALT   = 1 << 5,
+        _BUTTON_MODIFIER_MAX        = 1 << 6,
+        _BUTTON_MODIFIER_INVALID    = -EINVAL,
```

Comment:
i'd drop this too. YOu don't need an "invalid" state for your flags enum here. 

if you define an "invalid" state for a flags enum you really are supposed to check for it everytime you use variables of this type, but you don't really want that here, hence just drop this line too